numba/cuda/tests/cudapy/test_record_dtype.py

```diff
@@ -279,5 +282,57 @@ def _createSampleArrays(self):
         self.samplerec2darr = np.zeros(1, dtype=recordwith2darray)[0]
 
 
+@skip_on_cudasim('Structured array attr access not supported in simulator')
```

Comment:
The tests of getting and setting items for device records should be in `numba/cuda/tests/cudadrv/test_cuda_devicerecord.py` - this file contains tests for jitted functions operating on record / structured types.

Reply:
Moved.